processor/k8sattributesprocessor/e2e_test.go

```diff
@@ -503,9 +504,8 @@ func TestE2E_NamespacedRBAC(t *testing.T) {
 				"k8s.labels.app":               newExpectedValue(equal, "telemetrygen-"+testID+"-traces-deployment"),
 				"k8s.container.name":           newExpectedValue(equal, "telemetrygen"),
 				"container.image.name":         newExpectedValue(equal, "ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen"),
-				"container.image.repo_digests": newExpectedValue(regex, "ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen@sha256:[0-9a-fA-f]{64}"),
-				"container.image.tag":          newExpectedValue(equal, "latest"),
-				"container.id":                 newExpectedValue(exist, ""),
+				"container.image.repo_digests": newExpectedValue(regex, "ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen@sha256:[0-9a-fA-f]{64}"), "container.image.tag": newExpectedValue(shouldnotexist, ""),
```

Comment:
nit: could you please split these 2 fields in different lines? (same for the tests bellow)

Reply:
oops, must have missed that